src/scss/pages/_homePage.scss

```diff
@@ -0,0 +1,125 @@
+.Section {
```

Comment:
Remove the word **Section** for the classes

Reply:
👍🏼